website/blog/testing.md

```diff
@@ -0,0 +1,512 @@
+# Lower Test Toil With Better Local Testing
```

Comment:
It would be great if we can add a section/subsection on:
* testing options available today 
* challenges to do proper testing (e.g., integration test means to deploy in a staging environment  ... takes time, painful)

Once we guide the reader through the challenges to do testing, I think they will appreciate more what they can do with weaver.

=================

Another alternative is when we talk about our testing options, we can also talk briefly about alternatives (what people do today and how much more painful is to do that)